[DOC-1108] Release notes for Voyager v2025.12.1 (#29632)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-voyager/migrate/assess-migration.md

      --source-db-user ybvoyager \
      --source-db-password password \
      --source-db-name dbname \
      --export-dir /path/to/export/dir
  ```

  {{% /tab %}}
  {{% tab header="Config file" lang="config" %}}

   ```sh
   yb-voyager assess-migration --config-file <path-to-config-file>
   ```

   A sample source database configuration is as follows:

   ```yaml
   source:
     host: primary-host
     port: 5432
     user: ybvoyager
     password: password
     db-name: dbname
   ```

     {{% /tab %}}
   {{< /tabpane >}}

    Ensure that all provided endpoints are accessible as they are strictly validated.

    Voyager discovers replicas from the primary, attempts best-effort validation, and prompts you to include them. If validation fails (common in RDS, Aurora, Kubernetes, or when using internal IPs), you can either continue with primary-only assessment or manually specify replica endpoints.

- Manual specification.

>     Explicitly provide the read replica endpoints when automatic discovery fails, or if you want precise control.

Reviewer: ```suggestion
You can specify the read replica endpoints explicitly if automatic discovery fails, or if you want precise control.
```
